test/cljam/io/csi_test.clj

```diff
@@ -27,3 +27,30 @@
       (cio/file test-csi-file)
       (cio/as-url (cio/file test-csi-file))
       (cio/as-url (str (:uri server) "/csi/test.csi")))))
+
+(deftest calc-loffsets-test
+  (is (= (#'csi/calc-loffsets '(3) '({:beg 1 :end 2 :file-beg 1}
+                                     {:beg 5 :end 6 :file-beg 2}) 0)
+         {3 1}))
```

Comment:
Here is my understanding of l-offsets where `min-shift` = `0`(, `depth` = `2`)

```
     |                                      0    level0
bin: |               1               |     ...   level1
     | 9 | 10| 11| 12| 13| 14| 15| 16| 17| ...   level2

pos: |-1-|-2-|-3-|-4-|-5-|-6-|-7-|-8-|-9-|-...

data: <----->                      file-beg 1, bin 1
                      <----->      file-beg 2, bin 1

bin -> loffset
{0 1, 1 1, 9 1, 10 1, 13 2, 14 2}

beg -> loffset
{1 1, 2 1, 5 1, 6 1}

beg -> loffset for available bins only
{1 1}
```

I don't get the point of why we don't need `depth` argument and why we expect `{3 1}` as a result.
Would you elaborate on that?